### PR TITLE
Resolving Issue 6

### LINKED ISSUE #6: Create Sign In View

### PR DESCRIPTION
**Pull request by ekraihan**
This resolves #6

### CODE REVIEW

File: app/adapters/blueprint.js [hunk 1]
```diff
@@ -0,0 +1,69 @@
+import DS from 'ember-data';
```
Comment (ekraihan) [inline app/adapters/blueprint.js]: This is a JSON API adapter that pulls code from Dr. Hill's RESTAdapter

File: app/adapters/blueprint.js [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+import Ember from 'ember';
+
+/**
+ * This adaper users ember-cli-gatekeeper's code for adding a token to every
+ * request.
+ *
+ * NOTE: The majority of this code came from the ember-cli-gatekeeper repository here:
+ * https://github.com/onehilltech/ember-cli-gatekeeper/blob/9f6f59f901613430f7e1530554017f37c5b5d66f/addon/-lib/user/adapters/rest.js
+ */
+export default DS.JSONAPIAdapter.extend({
```
Comment (ekraihan) [inline app/adapters/blueprint.js:12]: This is a JSON API adapter that pulls code from Dr. Hill's RESTAdapter

File: app/adapters/application.js [hunk 3]
```diff
@@ -1,5 +1,5 @@
-import DS from 'ember-data';
+import {default as BlueprintAdapter} from './blueprint';
```
Comment (ekraihan) [inline app/adapters/application.js:1]: I made the base adapter extend the blueprint adapter I made below

File: app/adapters/user.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+import ApplicationAdapter from './application';
+
+
+export default ApplicationAdapter.extend({
```
Comment (ekraihan) [inline app/adapters/user.js:4]: Extend the base adapter and add the user id to the URL all POST requests

File: mirage/config.js [hunk 5]
```diff
@@ -1,7 +1,16 @@
...
 
   this.get('/users', (schema) => {
     return schema.users.all();
   });
+
+  // this.post('/users', (schema,req) => {
+  //   console.log(req);
+  // });
+
+  this.passthrough('http://165.227.76.52:5000/**');
```
Comment (ekraihan) [inline mirage/config.js:15]: If a route is requested but not found in mirage, let the request pass through to our API

**Review by JerebChase — APPROVED** [review]
Awesome

**Pull request merged**